api/.env

```diff
@@ -0,0 +1,15 @@
+# This is the port the API will be available at
+PORT=3001
+
+# Assuming that you started your MySQL container with the below command:
+# docker run --name mysql -e MYSQL_ROOT_PASSWORD=mysql-root-password -e MYSQL_DATABASE=my-database -d -p 3306:3306 mysql:latest
+DB_CLIENT=mysql2
+DB_HOST=localhost
+DB_PORT=3306
+DB_USER=root
+DB_PASSWORD=root
+DB_DATABASE_NAME=Meal
+
+# The database you use for local development most likely doesn't use SSL (encryption)
+# but the one you deploy on Render.com does.
+DB_USE_SSL=false
```

Comment:
This .env file shouldn't be pushed to GitHub. 
should be included in the` .gitignore` file